lib/services/ios-provision-service.ts

```diff
@@ -90,6 +103,31 @@ export class IOSProvisionService {
 
 		return { devices, match };
 	}
+
+	public async getDevelopmentTeams(): Promise<{ id: string, name: string }[]> {
+		const teams: { [teamName: string]: Set<string> } = {};
```

Comment:
This is used locally and just once. I don't want to pollute the module with interfaces. It could be IDictionary<Set<string>> but then it would be even harder to figure out what the key would be.